internal/activity.go

```diff
@@ -269,6 +270,13 @@ func WithActivityTask(
 	} else {
 		deadline = startToCloseDeadline
 	}
+
+	logger.With(
+		zapcore.Field{Key: tagWorkflowType, Type: zapcore.StringType, String: *task.WorkflowType.Name},
+		zapcore.Field{Key: tagWorkflowID, Type: zapcore.StringType, String: *task.WorkflowExecution.WorkflowId},
+		zapcore.Field{Key: tagRunID, Type: zapcore.StringType, String: *task.WorkflowExecution.RunId},
+	)
```

Comment:
Is this logging change intended to go with this PR?

Reply:
No, It was planning go with this pr: https://github.com/uber-go/cadence-client/pull/669
I was somehow checkout out along with this commit.

Reply:
I will remove this commit